examples/textarea/style.css

```diff
@@ -0,0 +1,57 @@
+.label {
+  display: flex;
+  width: auto;
+  flex-direction: column;
+  gap: 0.5rem;
+  text-indent: 0.5rem;
+}
+
+.textarea {
+  border-radius: 0.375rem;
+  border-style: none;
+  background-color: white;
+  padding: 0.5rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  color: black;
+  outline-width: 1px;
+  outline-offset: -1px;
+  outline-color: hsl(204deg 100% 40%);
+  box-shadow:
+    inset 0 0 0 1px rgba(0 0 0/15%),
+    inset 0 2px 5px 0 rgba(0 0 0/8%);
+  height: auto;
+  width: 320px;
+  field-sizing: contents;
+}
+
+.textarea::placeholder {
+  color: rgb(0 0 0 / 60%);
+}
+
+.textarea:hover {
+  background-color: hsl(204deg 20% 99%);
+}
+
+.textarea[data-active-item] {
+  outline-width: 2px;
+}
+
+.textarea:where(.dark, .dark *) {
+  background-color: hsl(204deg 4% 8%);
+  color: white;
+  box-shadow:
+    inset 0 0 0 1px rgba(255 255 255/15%),
+    inset 0 -1px 0 0 rgba(255 255 255/5%),
+    inset 0 2px 5px 0 rgba(0 0 0/15%);
+}
+
+.textarea:where(.dark, .dark *)::placeholder {
+  color: rgb(255 255 255 / 46%);
+}
+
+.textarea:hover:where(.dark, .dark *) {
+  background-color: hsl(204deg 4% 6%);
+}
```

Comment:
This should use Tailwind. See [Styling the example](https://github.com/ariakit/ariakit/blob/main/contributing.md#styling-the-example) and [Importing styles from other examples](https://github.com/ariakit/ariakit/blob/main/contributing.md#importing-styles-from-other-examples) in case you want to reuse existing styles from another example.

Reply:
https://github.com/ariakit/ariakit/pull/4032/commits/c0005c623b0ba923a6a5c2fc4d965556476f7b0a: copied the tailwind code from `combobox-textarea`. I couldn't import the file because they use different class names